staging/src/k8s.io/apiserver/pkg/authentication/token/cache/cache_test.go

```diff
@@ -48,8 +48,9 @@ func BenchmarkStripedCache(b *testing.B) {
 }
 
 func benchmarkCache(cache cache, b *testing.B) {
+	const numKeys = 1 << 16
 	keys := []string{}
-	for i := 0; i < b.N; i++ {
+	for i := 0; i < numKeys; i++ {
```

Comment:
I know I did a weird thing with this benchmark; the whole point was to measure the average time as a function of the number of unique keys being tested, to demonstrate that performance had an inflection point.

With this change that will be constant? What does the benchmark test now? I think there is a comment that needs to be changed to match the new behavior?

Reply:
I'm fine with doing that.

It's actually a 2 dimensional space, though-- tokens x qps.

Reply:
> run the benchmark at a variety of key counts

Can you do this before we merge, since measuring this is the purpose of the benchmark? Sorry for not being clear that I was waiting on this. It shouldn't be a big change from what you have, I think?

Reply:
@lavalamp, ya, I meant to poke you on #84408. Will make the change.